static/js/lib/util.js

```diff
@@ -2,6 +2,7 @@
 import type { Match } from "react-router"
 
 export const getChannelName = (props: { match: Match }): string => props.match.params.channelName || ""
+export const getPostID = (props: { match: Match }): string => props.match.params.postID || ""
```

Comment:
fair enough, I meant safe in the sense that if `props.match` is `undefined` then calling `.params` on it will throw an error, but I think you're right that in that case we would want it to fail loudly.